packages/sanity/src/core/form/inputs/arrays/ArrayOfObjectsInput/ArrayOfObjectsFunctions.tsx

```diff
@@ -51,18 +39,18 @@ export function ArrayOfObjectsFunctions<
       ? 'inputs.array.action.add-item-select-type'
       : 'inputs.array.action.add-item'
 
+  const insertButtonProps: React.ComponentProps<typeof Button> = {
+    icon: AddIcon,
+    mode: 'ghost',
+    size: 'large',
+    text: t(addItemI18nKey),
+  }
+
   if (readOnly) {
     return (
       <Tooltip portal content={t('inputs.array.read-only-label')}>
         <Grid>
-          <Button
-            icon={AddIcon}
-            mode="ghost"
-            disabled
-            size="large"
-            data-testid="add-read-object-button"
-            text={t(addItemI18nKey)}
-          />
+          <Button {...insertButtonProps} data-testid="add-read-object-button" disabled />
```

Comment:
The UI visibly looks the same so it might not make any difference, but since we'e removed the muted and such I just want to double check that we're not changing anything visually or if we are that it's been given the go ahead by design? :)

Reply:
Sorry about that, I just saw that it was just moved to the insertButtonProps that's why it was removed 🤦 don't mind me 🫥

Reply:
No worries :D It's just me trying to reuse props instead of copying them a third time 🙈 Let me know if it's confusing!

Reply:
Nope, they are super clear I'm just blind 😄 👓